bindings/python/setup.py

```diff
@@ -1,21 +1,32 @@
 import os
+import sys
 
 from setuptools import setup, find_packages
 
-# Our wheels are platform specific because we embed libmongocrypt.
+# Make our Windows and macOS wheels platform specific because we embed
+# libmongocrypt. On Linux we ship manylinux2010 wheels which cannot do this or
+# else auditwheel raises the following error:
+# RuntimeError: Invalid binary wheel, found the following shared
+# library/libraries in purelib folder:
+# 	libmongocrypt.so
+# The wheel has to be platlib compliant in order to be repaired by auditwheel.
 try:
     from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
-    class bdist_wheel(_bdist_wheel):
 
-        def finalize_options(self):
-            _bdist_wheel.finalize_options(self)
-            self.root_is_pure = False
+    if sys.platform in ('win32', 'darwin'):
```

Comment:
Why not put the entire try block inside of this if statement? You only need to import from wheel on those two platforms, nowhere else.

Reply:
Done.